warning: `QFontMetrics::width()` is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/MISC/GUIHelpers.cpp: In function ‘QRectF OpenMS::GUIHelpers::getTextDimension(const QStringList&, const QFont&, int&)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/MISC/GUIHelpers.cpp:195:48: warning: ‘int QFontMetrics::width(const QString&, int) const’ is deprecated: Use QFontMetrics::horizontalAdvance [-Wdeprecated-declarations]
  195 |       width = std::max(width, 4 + metrics.width(text[i]));
      |                                   ~~~~~~~~~~~~~^~~~~~~~~
In file included from /usr/include/qt/QtWidgets/qwidget.h:50,
                 from /usr/include/qt/QtWidgets/qdialog.h:44,
                 from /usr/include/qt/QtWidgets/qmessagebox.h:45,
                 from /usr/include/qt/QtWidgets/QMessageBox:1,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/MISC/GUIHelpers.cpp:42:
/usr/include/qt/QtGui/qfontmetrics.h:106:9: note: declared here
  106 |     int width(const QString &, int len = -1) const;
      |         ^~~~~
```

--- a/src/openms_gui/source/VISUAL/MISC/GUIHelpers.cpp
+++ b/src/openms_gui/source/VISUAL/MISC/GUIHelpers.cpp
@@ -192,7 +192,13 @@ namespace OpenMS
     int width = 4;
     for (int i = 0; i < text.size(); ++i)
     {
+      /*
+       * QFontMetrics::width() is deprecated after Qt 5.11. Use QFontMetrics::horizontalAdvance()
+       */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
       width = std::max(width, 4 + metrics.width(text[i]));
+#pragma GCC diagnostic pop
     }
     return QRectF(0, 0, width, height);
   }

